panic on watch-only CN

When the node is not elected and only watches the consensus process it does this on 0.77.0:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0xa58110]

goroutine 1035886 [running]:
github.com/nspcc-dev/neo-go/pkg/consensus.(*service).getStateRootSig(0xc000092280, 0x117, 0x3ee4f8b588e368f1, 0x3b9aca00)
        github.com/nspcc-dev/neo-go/pkg/consensus/consensus.go:257 +0x90
github.com/nspcc-dev/neo-go/pkg/consensus.(*service).newPrepareResponse(0xc000092280, 0x1, 0x1)
        github.com/nspcc-dev/neo-go/pkg/consensus/consensus.go:269 +0x11c
github.com/nspcc-dev/dbft.(*Context).makePrepareResponse(0xc0000ae280, 0x1016000, 0xc00013ec60)
        github.com/nspcc-dev/dbft@v0.0.0-20200810081309-f40804dbf8a0/send.go:94 +0x35
github.com/nspcc-dev/dbft.(*DBFT).sendPrepareResponse(0xc0000ae280)
        github.com/nspcc-dev/dbft@v0.0.0-20200810081309-f40804dbf8a0/send.go:104 +0x45
github.com/nspcc-dev/dbft.(*DBFT).onPrepareRequest(0xc0000ae280, 0x1019ec0, 0xc02d96a980)
        github.com/nspcc-dev/dbft@v0.0.0-20200810081309-f40804dbf8a0/dbft.go:319 +0xe67
github.com/nspcc-dev/dbft.(*DBFT).OnReceive(0xc0000ae280, 0x1019ec0, 0xc02d96a980)
        github.com/nspcc-dev/dbft@v0.0.0-20200810081309-f40804dbf8a0/dbft.go:233 +0xf40
github.com/nspcc-dev/neo-go/pkg/consensus.(*service).eventLoop(0xc000092280)
        github.com/nspcc-dev/neo-go/pkg/consensus/consensus.go:208 +0x72e
created by github.com/nspcc-dev/neo-go/pkg/consensus.(*service).Start
        github.com/nspcc-dev/neo-go/pkg/consensus/consensus.go:171 +0x9b
```
It shouldn't do so.

--- a/go.mod
+++ b/go.mod
@@ -8,7 +8,7 @@ require (
 	github.com/go-redis/redis v6.10.2+incompatible
 	github.com/gorilla/websocket v1.4.2
 	github.com/mr-tron/base58 v1.1.2
-	github.com/nspcc-dev/dbft v0.0.0-20200810081309-f40804dbf8a0
+	github.com/nspcc-dev/dbft v0.0.0-20200814063241-c789b8863620
 	github.com/nspcc-dev/rfc6979 v0.2.0
 	github.com/pkg/errors v0.8.1
 	github.com/prometheus/client_golang v1.2.1

